docs/cosmology/traits.rst

```diff
@@ -10,24 +10,26 @@ The :mod:`~astropy.cosmology.traits` module provides reusable components, called
 :term:`traits <trait type>`, that encapsulate specific cosmological properties or
 behaviors. For example, the :class:`~astropy.cosmology.traits.HubbleParameter` trait
 provides the Hubble constant (``H0``) and related methods, while
-:class:`~astropy.cosmology.traits.ScaleFactor` and
-:class:`~astropy.cosmology.traits.TemperatureCMB` provide the scale factor and the
-temperature or the CMB, respectively.
+:class:`~astropy.cosmology.traits.ScaleFactor`,
+:class:`~astropy.cosmology.traits.TemperatureCMB` and
+:class:`~astropy.cosmology.traits.DarkEnergyComponent` provide the scale factor, the
+temperature or the CMB, and the Dark Energy component, respectively.
 
 By combining these traits, you can easily construct custom cosmology classes with
 precisely the features you need, without having to reimplement common functionality.
 
 
 Here is an example of how to use the
 :class:`~astropy.cosmology.traits.HubbleParameter`,
-:class:`~astropy.cosmology.traits.ScaleFactor`, and
-:class:`~astropy.cosmology.traits.TemperatureCMB` traits in a custom cosmology class:
+:class:`~astropy.cosmology.traits.ScaleFactor`,
+:class:`~astropy.cosmology.traits.TemperatureCMB` and
+:class:`~astropy.cosmology.traits.DarkEnergyComponent` traits in a custom cosmology class:
 
 >>> import astropy.units as u
->>> from astropy.cosmology.traits import HubbleParameter, ScaleFactor, TemperatureCMB
+>>> from astropy.cosmology.traits import HubbleParameter, ScaleFactor, TemperatureCMB, DarkEnergyComponent
 >>> from astropy.cosmology import Cosmology
 >>>
->>> class CustomCosmology(Cosmology, HubbleParameter, ScaleFactor, TemperatureCMB):
+>>> class CustomCosmology(Cosmology, HubbleParameter, ScaleFactor, TemperatureCMB, DarkEnergyComponent):
```

Comment:
Now there are some DE specific methods. It would be great to showcase one below.

Reply:
I have tried to show a custom w(z) to show thats possible. I am thinking this is along the lines of what you were expecting.